nix/packages/treefmt/default.nix

```diff
@@ -42,8 +42,8 @@ in
     ldflags = [
       "-s"
       "-w"
-      "-X github.com/numtide/treefmt/build.Name=${pname}"
-      "-X github.com/numtide/treefmt/build.Version=v${version}"
+      "-X github.com/numtide/treefmt/v2/build.Name=${pname}"
```

Comment:
I'm not enough of a go expert to know of these were supposed to change.

Reply:
They were supposed to changed.